types/index.d.ts

```diff
@@ -340,21 +345,21 @@ declare namespace unified {
     /**
      * Transform an AST node/tree into text
      *
-     * @param node Node to be stringified
+     * @param node Node/tree to be stringified
      * @param file File associated with node
-     * @returns transformed text
+     * @returns Compiled text
      */
-    compile(node: Node, file?: VFileCompatible): string
+    compile(node: Node, file: VFile): string
```

Comment:
do these get passed to the compile method? looking at
https://github.com/unifiedjs/unified/blob/6c84a1f118eaed1606fa17a8d10f2bbef66d7ad3/index.js#L341
it looks like these params are passed to the class initializer rather than the compile method.